cleanlab/regression/rank.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+from cleanlab.outlier import OutOfDistribution
+from sklearn.neighbors import NearestNeighbors
+from cleanlab.internal.regression_utils import assert_valid_inputs
+from typing import Dict, Callable
+from numpy.typing import ArrayLike
```

Comment:
```suggestion
from typing import Dict, Callable
import numpy as np
from numpy.typing import ArrayLike
from sklearn.neighbors import NearestNeighbors

from cleanlab.outlier import OutOfDistribution
from cleanlab.internal.regression_utils import assert_valid_inputs
```

Reply:
[nit] better organization of dependencies

Reply:
Move the import of dependencies below the module docstring below and the copyright statement I inserted above. Look at how any other file is organized like cleanlab/rank.py

Reply:
handled here: [f4571b9](https://github.com/cleanlab/cleanlab/pull/572/commits/f4571b9a14b97c5c589bcd9371faf14adc004cd6)